mistralrs/examples/llava/main.rs

```diff
@@ -0,0 +1,76 @@
+use either::Either;
+use image::{ColorType, DynamicImage};
+use indexmap::IndexMap;
+use std::sync::Arc;
+use tokio::sync::mpsc::channel;
+
+use mistralrs::{
+    Constraint, Device, DeviceMapMetadata, MistralRs, MistralRsBuilder, ModelDType, NormalRequest,
+    Request, RequestMessage, Response, SamplingParams, SchedulerMethod, TokenSource,
+    VisionLoaderBuilder, VisionLoaderType, VisionSpecificConfig,
+};
+
+fn setup() -> anyhow::Result<Arc<MistralRs>> {
+    let loader = VisionLoaderBuilder::new(
+        VisionSpecificConfig {
+            use_flash_attn: false,
+            repeat_last_n: 64,
+        },
+        Some("chat_templates/vicuna.json".to_string()),
+        None,
+        Some("llava-hf/llava-1.5-7b-hf".to_string()),
+    )
+    .build(VisionLoaderType::LLaVA);
+    // Load, into a Pipeline
+
+    let pipeline = loader.load_model_from_hf(
+        None,
+        TokenSource::CacheToken,
+        &ModelDType::Auto,
+        &Device::cuda_if_available(0)?,
+        false,
+        DeviceMapMetadata::dummy(),
+        None,
+    )?;
+    // Create the MistralRs, which is a runner
+    Ok(MistralRsBuilder::new(pipeline, SchedulerMethod::Fixed(5.try_into().unwrap())).build())
+}
+
+fn main() -> anyhow::Result<()> {
+    let mistralrs = setup()?;
+
+    let (tx, mut rx) = channel(10_000);
+    let request = Request::Normal(NormalRequest {
+        messages: RequestMessage::VisionChat {
+            images: vec![DynamicImage::new(1280, 720, ColorType::Rgb8)],
+            messages: vec![IndexMap::from([
+                ("role".to_string(), Either::Left("user".to_string())),
+                (
+                    "content".to_string(),
+                    Either::Left("<|image_1|>what is this image show?".to_string()),
```

Comment:
Same comment as above, I think this should be `<image>` as per the HF model card.